modules/openid_auth/app/serializers/openid_auth/validation_serializer.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module OpenidAuth
+  class ValidationSerializer < ActiveModel::Serializer
+    alias read_attribute_for_serialization send
+
+    type 'validated_token'
+
+    def id
+      object.jti
+    end
+
+    attributes :ver,
+               :jti,
+               :iss,
+               :aud,
+               :iat,
+               :exp,
+               :cid,
+               :uid,
+               :scp,
+               :sub,
+               :va_identifiers
+
+    delegate :ver, to: :object
```

Comment:
I think these can be written as `delegate :ver, :jti, {..list of all the methose}, to: :object`

Reply:
Thanks for updating